app-editors/vis/vis-0.3.ebuild

```diff
@@ -17,7 +17,8 @@ IUSE="+ncurses selinux test tre"
 
 #Note: vis is reported to also work with NetBSD curses
 #TODO: >=dev-lang/lua-5.2 (needed for syntax highlighting and settings)
-DEPEND=" dev-libs/libtermkey
+DEPEND=" app-eselect/eselect-vi
```

Comment:
Is it really a build-time dependency? I dare say it's only needed at runtime and `pkg_post*` which is also covered by runtime deps.